scanning/static/js/test_generator.js

```diff
@@ -0,0 +1,81 @@
+const TestGenerator = (() => {
```

Comment:
![image](https://user-images.githubusercontent.com/26633722/82764927-9e29dc00-9de0-11ea-8731-2d6d79393936.png)

When I tried testing out the generator, the accounts are created successfully but the qr codes didn't appear on the scan page. After looking at the console, looks like I'm missing the qrcode_generator.js file -- is this something that needs to be fixed or how long does it take for the codes to appear after clicking generate?